libdnf/transaction/MergedTransaction.cpp

```diff
@@ -171,6 +172,21 @@ MergedTransaction::getConsoleOutput()
     return output;
 }
 
+
+static bool transaction_item_sort_function(const std::shared_ptr<TransactionItemBase> one, const std::shared_ptr<TransactionItemBase> two) {
+    if (one->isForwardAction() && one->isForwardAction()) {
+        return true;
+    }
+    if (one->isBackwardAction() && one->isBackwardAction()) {
```

Comment:
I uses two same arguments. Please could you verify it?

Reply:
The same as above; I've fixed both.